docs/use/data-flows.md

```diff
@@ -1,80 +1,81 @@
-# Getting Started With CodeSee: Anatomy of a Data Flow
+# Getting started With CodeSee: anatomy of a data flow
```

Comment:
Just to be sure, if Data Flows are capitalised as a CodeSee-specific thing, should they be capitalised here too?

Reply:
Ahh . . . yes. Ok if Data Flows (and Stories) are capitalized product things, I need to do a bunch of fixes . . . brb